build/templates/rep_caps.rst.mako

```diff
@@ -7,6 +7,7 @@
 
 %>\
 .. py:module:: ${module_name}
+    :noindex:
```

Comment:
What does this do?

Reply:
`.. py:module:: <module_name>` is also at the top of class.rst. Only one instance can be in the index, and I figured the class instance was more important.

This gets rid of a warning from Sphinx when generating the html files.